config/environments/production.rb

```diff
@@ -16,71 +14,57 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
-  # Enable Rack::Cache to put a simple HTTP cache in front of your application
-  # Add `rack-cache` to your Gemfile before enabling this.
-  # For large-scale production use, consider using a caching reverse proxy like nginx, varnish or squid.
-  # config.action_dispatch.rack_cache = true
-
-  # action mailer config -- required for password resets and the bulk uploader
-    config.action_mailer.smtp_settings = {
-    :address   => "smtp.yourdomain.com",
-    :port      => 587, # ports 587 and 2525 are also supported with STARTTLS
-    :enable_starttls_auto => true, # detects and uses STARTTLS
-    :user_name => "XXXXX",
-    :password  => "XXXXX",
-    :authentication => 'login',
-    :domain => 'yourdomain.com', # your domain to identify your server when connecting
-  }
-  config.action_mailer.default_url_options =  { host: 'hostname' } #change this to match your server URL, i.e. www.fromthepage.com
-  config.action_mailer.default_options
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
-  
```

Comment:
Where did action_mailer config move?  We need it.

Reply:
@saracarl I just restored that config.